README.md

```diff
@@ -72,3 +72,9 @@
 # Courses
 
 [Test-Driven Development with Python, Flask, and Docker](https://testdriven.io/courses/tdd-flask/)
+
+# Books
+
+Python Testing with pytest by Brian Okken - [The Pragmatic Programmers (publisher)](https://pragprog.com/book/bopytest/python-testing-with-pytest) | [Amazon](https://www.amazon.com/Python-Testing-pytest-Effective-Scalable/dp/1680502409)
+
+pytest Quick Start Guide by Bruno Oliveira - [Packt (publisher)](https://www.packtpub.com/web-development/pytest-quick-start-guide) | [Amazon](https://www.amazon.com/pytest-Quick-Start-Guide-maintainable/dp/1789347564)
```

Comment:
I'd prefer to keep only the publisher's link here. Can you please remove the Amazon links?